samples/suit/smp_transfer/README.rst

```diff
@@ -268,91 +277,205 @@ After programming the sample to your development kit and updating the sequence n
+                     digest: f391a4e298e0fc76d3b5d3044e379255484a2afde5e57d03dee5e07c9b4f558d
+                     digestAlgorithm: sha256
+                     signatureCheck: signature check not performed
+                     sequenceNumber: 1
+                     semantic version: 0.1.0
+
+         #. Upload the image with nrfutil:
 
             .. code-block:: console
 
-               mcumgr --conntype serial --connstring "dev=/dev/ttyACM0,baud=115200" image upload build/DFU/root.suit
+               nrfutil suit upload-envelope --serial-port=/dev/ttyACM0 --envelope-file build/DFU/root.suit
 
             You should see an output similar to the following logged on UART:
 
-            .. parsed-literal::
-               :class: highlight
-
-               0 / 250443 [---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------]   0.00%
-               18.99 KiB / 244.57 KiB [============>-------------------------------------------------------------------------------------------------------------------------------------------------]   7.76% 11.83 KiB/s 00m19s
-               66.56 KiB / 244.57 KiB [==========================================>-------------------------------------------------------------------------------------------------------------------]  27.21% 18.44 KiB/s 00m09s
-               112.12 KiB / 244.57 KiB [=======================================================================>-------------------------------------------------------------------------------------]  45.84% 19.97 KiB/s 00m06s
-               154.08 KiB / 244.57 KiB [==================================================================================================>----------------------------------------------------------]  63.00% 20.22 KiB/s 00m04s
-               197.40 KiB / 244.57 KiB [==============================================================================================================================>------------------------------]  80.71% 20.51 KiB/s 00m02s
-               241.16 KiB / 244.57 KiB [=================================================================================================================================================================>--]  98.60% 20.74 KiB/s
-               Done
+            .. code-block:: console
+
+               [00:00:07] ###### 100% [/dev/ttyACM0] Uploaded
+
       #. If you have built the application with :ref:`external flash support <nrf54h_suit_sample_extflash>`, upload the cache partition to the external flash using the following command:
 
          .. code-block:: console
 
-            mcumgr --conntype serial --connstring "dev=/dev/ttyACM0,baud=115200" image upload -n 2 build/DFU/dfu_cache_partition_1.bin
+            nrfutil suit upload-cache-raw --serial-port=/dev/ttyACM0 --cache-file build/dfu_cache_partition_1.bin --pool 2
```

Comment:
```suggestion
            nrfutil suit upload-cache-raw --serial-port COM4 --cache-file build/dfu_cache_partition_1.bin --pool 2
```

Reply:
Will change to COM7 like in nrfutil docs